content/en/docs/instrumentation/java/manual.md

```diff
@@ -348,6 +348,70 @@ public void handle(HttpExchange httpExchange) {
   }
 }
 ```
+The following code presents an example of fetching the incoming context propagated, add spans, and further propagate the context. The example utilizes [HttpHeaders](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpHeaders.html) to fetch the traceparent header for context propagation.
```

Comment:
Hmmm. I don't really understand this sentence. What does "fetching the incoming context propagated" mean?

Reply:
What I was trying to convey was that the trace contains some preexisting spans